2016-06-07 17:39:31 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "PATH" not found; SQL statement:
select * from path where id_bpmn = 'asd'; [42102-191]
2016-06-10 18:46:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "'a23748293649' (PATH: ID_BPMN INTEGER NOT NULL)"; SQL statement:
insert into glimpse.path (id_bpmn, absolute_session_score, path_rule) values (?, ?, ?) -- (?1, ?2, ?3) [22018-191]
2016-06-10 18:46:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "'a23748293649' (PATH: ID_BPMN INTEGER NOT NULL)"; SQL statement:
insert into glimpse.path (id_bpmn, absolute_session_score, path_rule) values (?, ?, ?) -- (?1, ?2, ?3) [22018-191]
2016-06-10 18:46:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "'a23748293649' (PATH: ID_BPMN INTEGER NOT NULL)"; SQL statement:
insert into glimpse.path (id_bpmn, absolute_session_score, path_rule) values (?, ?, ?) -- (?1, ?2, ?3) [22018-191]
2016-06-10 18:46:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "'a23748293649' (BPMN: ID_BPMN INTEGER NOT NULL)"; SQL statement:
insert into glimpse.bpmn (id_bpmn, extraction_date, id_category, absolute_bp_score, paths_cardinality) values (?, ?, ?, ?, ?) -- (?1, ?2, ?3, ?4, ?5) [22018-191]
06-14 19:38:12 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 19:38:12 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 19:38:49 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Table "LEARNER" not found; SQL statement:
select * from learner [42102-176]
2016-06-14 20:05:35 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "'a23748293649' (PATH: ID_BPMN INTEGER NOT NULL)"; SQL statement:
insert into glimpse.path (id_bpmn, absolute_session_score, path_rule) values (?, ?, ?) -- (?1, ?2, ?3) [22018-191]
2016-06-14 20:05:35 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "'a23748293649' (PATH: ID_BPMN INTEGER NOT NULL)"; SQL statement:
insert into glimpse.path (id_bpmn, absolute_session_score, path_rule) values (?, ?, ?) -- (?1, ?2, ?3) [22018-191]
2016-06-14 20:05:35 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "'a23748293649' (PATH: ID_BPMN INTEGER NOT NULL)"; SQL statement:
insert into glimpse.path (id_bpmn, absolute_session_score, path_rule) values (?, ?, ?) -- (?1, ?2, ?3) [22018-191]
2016-06-14 20:05:55 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "'a23748293649' (BPMN: ID_BPMN INTEGER NOT NULL)"; SQL statement:
insert into glimpse.bpmn (id_bpmn, extraction_date, id_category, absolute_bp_score, paths_cardinality) values (?, ?, ?, ?, ?) -- (?1, ?2, ?3, ?4, ?5) [22018-191]
06-14 20:06:32 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 20:06:32 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 20:06:32 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 20:06:32 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 20:06:32 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 20:06:32 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 20:07:50 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 20:07:50 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 20:07:50 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 20:07:50 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 20:08:33 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 20:08:33 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 20:08:33 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 20:08:33 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
2016-06-14 20:09:01 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "'a23748293649' (PATH: ID_BPMN INTEGER NOT NULL)"; SQL statement:
insert into glimpse.path (id_bpmn, absolute_session_score, path_rule) values (?, ?, ?) -- (?1, ?2, ?3) [22018-191]
2016-06-14 20:09:01 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "'a23748293649' (PATH: ID_BPMN INTEGER NOT NULL)"; SQL statement:
insert into glimpse.path (id_bpmn, absolute_session_score, path_rule) values (?, ?, ?) -- (?1, ?2, ?3) [22018-191]
2016-06-14 20:09:01 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "'a23748293649' (PATH: ID_BPMN INTEGER NOT NULL)"; SQL statement:
insert into glimpse.path (id_bpmn, absolute_session_score, path_rule) values (?, ?, ?) -- (?1, ?2, ?3) [22018-191]
2016-06-14 20:09:01 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "'a23748293649' (BPMN: ID_BPMN INTEGER NOT NULL)"; SQL statement:
insert into glimpse.bpmn (id_bpmn, extraction_date, id_category, absolute_bp_score, paths_cardinality) values (?, ?, ?, ?, ?) -- (?1, ?2, ?3, ?4, ?5) [22018-191]
2016-06-14 20:12:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "'a23748293649' (PATH: ID_BPMN INTEGER NOT NULL)"; SQL statement:
insert into glimpse.path (id_bpmn, absolute_session_score, path_rule) values (?, ?, ?) -- (?1, ?2, ?3) [22018-191]
2016-06-14 20:12:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "'a23748293649' (PATH: ID_BPMN INTEGER NOT NULL)"; SQL statement:
insert into glimpse.path (id_bpmn, absolute_session_score, path_rule) values (?, ?, ?) -- (?1, ?2, ?3) [22018-191]
2016-06-14 20:12:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "'a23748293649' (PATH: ID_BPMN INTEGER NOT NULL)"; SQL statement:
insert into glimpse.path (id_bpmn, absolute_session_score, path_rule) values (?, ?, ?) -- (?1, ?2, ?3) [22018-191]
2016-06-14 20:12:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "'a23748293649' (BPMN: ID_BPMN INTEGER NOT NULL)"; SQL statement:
insert into glimpse.bpmn (id_bpmn, extraction_date, id_category, absolute_bp_score, paths_cardinality) values (?, ?, ?, ?, ?) -- (?1, ?2, ?3, ?4, ?5) [22018-191]
2016-06-14 20:13:36 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "'a23748293649' (PATH: ID_BPMN INTEGER NOT NULL)"; SQL statement:
insert into glimpse.path (id_bpmn, absolute_session_score, path_rule) values (?, ?, ?) -- (?1, ?2, ?3) [22018-191]
2016-06-14 20:13:36 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "'a23748293649' (PATH: ID_BPMN INTEGER NOT NULL)"; SQL statement:
insert into glimpse.path (id_bpmn, absolute_session_score, path_rule) values (?, ?, ?) -- (?1, ?2, ?3) [22018-191]
2016-06-14 20:13:36 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "'a23748293649' (PATH: ID_BPMN INTEGER NOT NULL)"; SQL statement:
insert into glimpse.path (id_bpmn, absolute_session_score, path_rule) values (?, ?, ?) -- (?1, ?2, ?3) [22018-191]
2016-06-14 20:13:36 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "'a23748293649' (BPMN: ID_BPMN INTEGER NOT NULL)"; SQL statement:
insert into glimpse.bpmn (id_bpmn, extraction_date, id_category, absolute_bp_score, paths_cardinality) values (?, ?, ?, ?, ?) -- (?1, ?2, ?3, ?4, ?5) [22018-191]
2016-06-14 20:14:03 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "'a23748293649' (PATH: ID_BPMN INTEGER NOT NULL)"; SQL statement:
insert into glimpse.path (id_bpmn, absolute_session_score, path_rule) values (?, ?, ?) -- (?1, ?2, ?3) [22018-191]
2016-06-14 20:14:03 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "'a23748293649' (PATH: ID_BPMN INTEGER NOT NULL)"; SQL statement:
insert into glimpse.path (id_bpmn, absolute_session_score, path_rule) values (?, ?, ?) -- (?1, ?2, ?3) [22018-191]
2016-06-14 20:14:03 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "'a23748293649' (PATH: ID_BPMN INTEGER NOT NULL)"; SQL statement:
insert into glimpse.path (id_bpmn, absolute_session_score, path_rule) values (?, ?, ?) -- (?1, ?2, ?3) [22018-191]
2016-06-14 20:14:03 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "'a23748293649' (BPMN: ID_BPMN INTEGER NOT NULL)"; SQL statement:
insert into glimpse.bpmn (id_bpmn, extraction_date, id_category, absolute_bp_score, paths_cardinality) values (?, ?, ?, ?, ?) -- (?1, ?2, ?3, ?4, ?5) [22018-191]
2016-06-14 20:23:56 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "'a23748293649' (PATH: ID_BPMN INTEGER NOT NULL)"; SQL statement:
insert into glimpse.path (id, id_bpmn, absolute_session_score, path_rule) values (?, ?, ?, ?) -- (?1, ?2, ?3, ?4) [22018-191]
2016-06-14 20:23:56 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "'a23748293649' (PATH: ID_BPMN INTEGER NOT NULL)"; SQL statement:
insert into glimpse.path (id, id_bpmn, absolute_session_score, path_rule) values (?, ?, ?, ?) -- (?1, ?2, ?3, ?4) [22018-191]
2016-06-14 20:23:56 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "'a23748293649' (PATH: ID_BPMN INTEGER NOT NULL)"; SQL statement:
insert into glimpse.path (id, id_bpmn, absolute_session_score, path_rule) values (?, ?, ?, ?) -- (?1, ?2, ?3, ?4) [22018-191]
2016-06-14 20:23:56 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "'a23748293649' (BPMN: ID_BPMN INTEGER NOT NULL)"; SQL statement:
insert into glimpse.bpmn (id_bpmn, extraction_date, id_category, absolute_bp_score, paths_cardinality) values (?, ?, ?, ?, ?) -- (?1, ?2, ?3, ?4, ?5) [22018-191]
2016-06-14 20:29:17 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "'a23748293649' (PATH: ID_BPMN INTEGER NOT NULL)"; SQL statement:
insert into glimpse.path (id, id_bpmn, absolute_session_score, path_rule) values (?, ?, ?, ?) -- (?1, ?2, ?3, ?4) [22018-191]
06-14 20:29:49 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 20:29:49 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 20:29:49 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 20:29:49 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 20:31:12 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 20:31:12 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 20:31:12 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 20:31:12 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 20:32:02 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 20:32:02 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 20:32:02 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 20:32:02 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 20:43:08 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 20:43:08 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 20:43:08 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 20:43:08 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 20:43:08 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 20:43:08 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 21:14:30 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 21:14:30 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 21:14:30 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 21:14:30 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 21:14:30 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 21:14:30 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 21:25:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 21:25:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 21:25:38 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 21:25:38 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 21:25:38 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 21:25:38 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 21:49:44 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 21:49:44 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 21:49:44 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 21:49:44 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 21:49:44 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 21:49:44 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
2016-06-14 21:50:28 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "'a23748293649-0' (PATH: ID INTEGER NOT NULL)"; SQL statement:
insert into glimpse.path (id, id_bpmn, absolute_session_score, path_rule) values (?, ?, ?, ?) -- (?1, ?2, ?3, ?4) [22018-191]
06-14 21:52:59 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 21:52:59 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 21:53:00 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 21:53:00 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 21:58:35 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 21:58:35 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 21:59:50 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 21:59:50 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 21:59:50 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 21:59:50 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 21:59:50 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 21:59:50 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
2016-06-14 22:00:49 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column "ID" not found; SQL statement:
 insert into glimpse.path (id, id_bpmn, absolute_session_score, path_rule) values (?, ?, ?, ?) [42122-191]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.table.Table.getColumn(Table.java:680)
	at org.h2.command.Parser.parseColumn(Parser.java:866)
	at org.h2.command.Parser.parseColumnList(Parser.java:850)
	at org.h2.command.Parser.parseInsert(Parser.java:1050)
	at org.h2.command.Parser.parsePrepared(Parser.java:411)
	at org.h2.command.Parser.parse(Parser.java:315)
	at org.h2.command.Parser.parse(Parser.java:287)
	at org.h2.command.Parser.prepareCommand(Parser.java:252)
	at org.h2.engine.Session.prepareLocal(Session.java:560)
	at org.h2.engine.Session.prepareCommand(Session.java:501)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1188)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:73)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:276)
	at eu.learnpad.simulator.mon.controller.H2Controller.savePath(H2Controller.java:211)
	at eu.learnpad.simulator.mon.controller.H2Controller.savePathsForBPMN(H2Controller.java:397)
	at eu.learnpad.simulator.mon.impl.LearnerAssessmentManagerImpl.elaborateModel(LearnerAssessmentManagerImpl.java:91)
	at eu.learnpad.simulator.mon.manager.GlimpseManager.onMessage(GlimpseManager.java:153)
	at org.apache.activemq.ActiveMQMessageConsumer.dispatch(ActiveMQMessageConsumer.java:1321)
	at org.apache.activemq.ActiveMQSessionExecutor.dispatch(ActiveMQSessionExecutor.java:131)
	at org.apache.activemq.ActiveMQSessionExecutor.iterate(ActiveMQSessionExecutor.java:202)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:129)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:47)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
2016-06-14 22:00:58 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Duplicate column name "ID_BPMN"; SQL statement:
 insert into glimpse.path (id_bpmn, id_bpmn, absolute_session_score, path_rule) values (?, ?, ?, ?) [42121-191]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.Parser.parseColumnList(Parser.java:852)
	at org.h2.command.Parser.parseInsert(Parser.java:1050)
	at org.h2.command.Parser.parsePrepared(Parser.java:411)
	at org.h2.command.Parser.parse(Parser.java:315)
	at org.h2.command.Parser.parse(Parser.java:287)
	at org.h2.command.Parser.prepareCommand(Parser.java:252)
	at org.h2.engine.Session.prepareLocal(Session.java:560)
	at org.h2.engine.Session.prepareCommand(Session.java:501)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1188)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:73)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:276)
	at eu.learnpad.simulator.mon.controller.H2Controller.savePath(H2Controller.java:211)
	at eu.learnpad.simulator.mon.controller.H2Controller.savePathsForBPMN(H2Controller.java:397)
	at eu.learnpad.simulator.mon.impl.LearnerAssessmentManagerImpl.elaborateModel(LearnerAssessmentManagerImpl.java:91)
	at eu.learnpad.simulator.mon.manager.GlimpseManager.onMessage(GlimpseManager.java:153)
	at org.apache.activemq.ActiveMQMessageConsumer.dispatch(ActiveMQMessageConsumer.java:1321)
	at org.apache.activemq.ActiveMQSessionExecutor.dispatch(ActiveMQSessionExecutor.java:131)
	at org.apache.activemq.ActiveMQSessionExecutor.iterate(ActiveMQSessionExecutor.java:202)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:129)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:47)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
2016-06-14 22:02:10 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column "id" not found [42122-191]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3129)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3217)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:303)
	at eu.learnpad.simulator.mon.controller.H2Controller.getBPMNPaths(H2Controller.java:71)
	at eu.learnpad.simulator.mon.impl.LearnerAssessmentManagerImpl.elaborateModel(LearnerAssessmentManagerImpl.java:98)
	at eu.learnpad.simulator.mon.manager.GlimpseManager.onMessage(GlimpseManager.java:153)
	at org.apache.activemq.ActiveMQMessageConsumer.dispatch(ActiveMQMessageConsumer.java:1321)
	at org.apache.activemq.ActiveMQSessionExecutor.dispatch(ActiveMQSessionExecutor.java:131)
	at org.apache.activemq.ActiveMQSessionExecutor.iterate(ActiveMQSessionExecutor.java:202)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:129)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:47)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
06-14 22:09:07 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "DELETE
FROM
    ""GLIMPSE"".""BPMN""
    ""GLIMPSE"".[*]""PATH""
    ""GLIMPSE"".""LEARNER"""; SQL statement:
DELETE
FROM
    "GLIMPSE"."BPMN"
    "GLIMPSE"."PATH"
    "GLIMPSE"."LEARNER" [42000-176]
06-14 22:09:17 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "DELETE
FROM
    ""GLIMPSE"".""BPMN""
DELETE
FROM[*]
    ""GLIMPSE"".""PATH""
DELETE
FROM
    ""GLIMPSE"".""LEARNER"""; SQL statement:
DELETE
FROM
    "GLIMPSE"."BPMN"
DELETE
FROM
    "GLIMPSE"."PATH"
DELETE
FROM
    "GLIMPSE"."LEARNER" [42000-176]
06-14 22:09:41 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 22:09:41 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 22:09:41 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 22:09:41 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 22:14:33 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 22:14:33 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 22:14:33 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 22:14:33 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 22:16:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 22:16:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 22:16:57 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 22:16:57 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 22:18:04 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 22:18:04 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 22:18:04 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 22:18:04 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 22:22:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 22:22:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 22:22:00 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 22:22:00 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 22:29:25 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 22:29:25 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 22:29:25 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 22:29:25 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 22:32:04 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 22:32:04 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 22:32:04 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-14 22:32:04 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 10:53:32 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 10:53:32 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 10:53:32 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 10:53:32 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 10:59:17 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 10:59:17 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 10:59:17 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 10:59:17 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 11:02:42 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 11:02:42 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 11:02:42 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 11:02:42 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 11:03:26 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 11:03:26 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 11:03:26 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 11:03:26 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 11:04:22 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 11:04:22 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 11:04:22 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 11:04:22 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 11:10:25 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 11:10:25 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 11:10:25 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 11:10:25 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 11:14:46 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 11:14:46 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 11:14:46 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 11:14:46 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 12:07:04 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 12:07:04 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 12:07:04 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 12:07:04 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 12:20:08 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 12:20:08 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 12:20:08 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 12:20:08 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 12:21:59 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 12:21:59 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 12:22:00 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 12:22:00 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 12:24:56 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 12:24:56 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 12:24:56 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 12:24:56 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
2016-06-15 12:25:02 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: IO Exception: "java.io.EOFException"; "/home/acalabro/workspaceLearnPAd/learnpad/lp-simulation-environment/monitoring/data/glimpse.h2.db"; SQL statement:
select * from glimpse.path where id_bpmn = 'a23748293649'; [90031-191]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convertIOException(DbException.java:330)
	at org.h2.store.FileStore.readFully(FileStore.java:275)
	at org.h2.store.PageStore.readPage(PageStore.java:1325)
	at org.h2.store.PageStore.getPage(PageStore.java:749)
	at org.h2.index.PageDataIndex.getPage(PageDataIndex.java:232)
	at org.h2.index.PageDataNode.find(PageDataNode.java:166)
	at org.h2.index.PageDataIndex.find(PageDataIndex.java:283)
	at org.h2.index.BaseIndex.find(BaseIndex.java:132)
	at org.h2.index.IndexCursor.find(IndexCursor.java:169)
	at org.h2.table.TableFilter.next(TableFilter.java:460)
	at org.h2.command.dml.Select.queryFlat(Select.java:541)
	at org.h2.command.dml.Select.queryWithoutCache(Select.java:654)
	at org.h2.command.dml.Query.query(Query.java:341)
	at org.h2.command.dml.Query.query(Query.java:309)
	at org.h2.command.dml.Query.query(Query.java:36)
	at org.h2.command.CommandContainer.query(CommandContainer.java:110)
	at org.h2.command.Command.executeQuery(Command.java:201)
	at org.h2.jdbc.JdbcPreparedStatement.executeQuery(JdbcPreparedStatement.java:110)
	at eu.learnpad.simulator.mon.controller.H2Controller.checkIfBPHasBeenAlreadyExtracted(H2Controller.java:284)
	at eu.learnpad.simulator.mon.impl.LearnerAssessmentManagerImpl.elaborateModel(LearnerAssessmentManagerImpl.java:78)
	at eu.learnpad.simulator.mon.manager.GlimpseManager.onMessage(GlimpseManager.java:155)
	at org.apache.activemq.ActiveMQMessageConsumer.dispatch(ActiveMQMessageConsumer.java:1321)
	at org.apache.activemq.ActiveMQSessionExecutor.dispatch(ActiveMQSessionExecutor.java:131)
	at org.apache.activemq.ActiveMQSessionExecutor.iterate(ActiveMQSessionExecutor.java:202)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:129)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:47)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.io.EOFException
	at org.h2.store.fs.FileUtils.readFully(FileUtils.java:358)
	at org.h2.store.FileStore.readFully(FileStore.java:273)
	... 27 more
06-15 12:25:25 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 12:25:25 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 12:25:25 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 12:25:25 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 12:26:42 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 12:26:42 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 12:26:42 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 12:26:42 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 12:27:59 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 12:27:59 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 12:27:59 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 12:27:59 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 12:32:18 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 12:32:18 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 12:32:18 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 12:32:18 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 12:35:42 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 12:35:42 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 12:35:42 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 12:35:42 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 12:37:42 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 12:37:42 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 12:37:42 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 12:37:42 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 12:44:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 12:44:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 12:44:16 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 12:44:16 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 12:46:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 12:46:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 12:46:13 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 12:46:13 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 12:50:34 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 12:50:34 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 12:50:34 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 12:50:34 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 12:54:36 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 12:54:36 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 12:54:36 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 12:54:36 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 13:01:27 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 13:01:27 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 13:01:27 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 13:01:27 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 13:08:35 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 13:08:35 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 13:08:35 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 13:08:35 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 13:15:44 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 13:15:44 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 13:15:45 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 13:15:45 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 16:26:04 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 16:26:04 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 16:26:04 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-15 16:26:04 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
2016-06-15 23:41:28 database: close
org.h2.message.DbException: IO Exception: "java.io.EOFException"; "/home/acalabro/workspaceLearnPAd/learnpad/lp-simulation-environment/monitoring/data/glimpse.h2.db" [90031-191]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convertIOException(DbException.java:330)
	at org.h2.store.FileStore.readFully(FileStore.java:275)
	at org.h2.store.PageStore.readPage(PageStore.java:1325)
	at org.h2.store.PageStore.getPage(PageStore.java:749)
	at org.h2.store.PageStreamTrunk$Iterator.next(PageStreamTrunk.java:251)
	at org.h2.store.PageLog.free(PageLog.java:213)
	at org.h2.store.PageStore.compact(PageStore.java:498)
	at org.h2.engine.Database.closeOpenFilesAndUnlock(Database.java:1356)
	at org.h2.engine.Database.close(Database.java:1286)
	at org.h2.engine.DatabaseCloser.run(DatabaseCloser.java:63)
Caused by: org.h2.jdbc.JdbcSQLException: IO Exception: "java.io.EOFException"; "/home/acalabro/workspaceLearnPAd/learnpad/lp-simulation-environment/monitoring/data/glimpse.h2.db" [90031-191]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 11 more
Caused by: java.io.EOFException
	at org.h2.store.fs.FileUtils.readFully(FileUtils.java:358)
	at org.h2.store.FileStore.readFully(FileStore.java:273)
	... 8 more
06-16 12:23:31 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-16 12:23:31 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-16 12:23:31 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-16 12:23:31 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-16 12:29:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-16 12:29:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-16 12:29:57 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-16 12:29:57 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-16 12:31:46 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "DISCONNECT[*] "; expected "DELETE, DROP, DECLARE, DEALLOCATE, {"; SQL statement:
Disconnect [42001-176]
06-16 12:31:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-16 12:31:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-16 12:31:58 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-16 12:31:58 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-16 12:32:03 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-16 12:32:03 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-16 12:32:03 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-16 12:32:03 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-16 12:46:14 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-16 12:46:14 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-16 12:46:14 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-16 12:46:14 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-16 12:47:22 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-16 12:47:22 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-16 12:47:22 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-16 12:47:22 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-16 13:01:36 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-16 13:01:36 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-16 13:01:36 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-16 13:01:36 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-16 13:06:49 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-16 13:06:49 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-16 13:06:49 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-16 13:06:49 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-16 13:10:36 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-16 13:10:36 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-16 13:10:36 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-16 13:10:36 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-16 13:18:27 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-16 13:18:27 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-16 13:18:27 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-16 13:18:27 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-16 13:26:55 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-16 13:26:55 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-16 13:26:55 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-16 13:26:55 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-16 13:31:31 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-16 13:31:31 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-16 13:31:31 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-16 13:31:31 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-16 13:36:12 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-16 13:36:12 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-16 13:36:12 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-16 13:36:12 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-16 13:38:05 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-16 13:38:05 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-16 13:38:05 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-16 13:38:05 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-16 17:51:56 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-16 17:51:56 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-20 12:54:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-20 12:54:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-20 12:54:24 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-20 12:54:24 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-20 17:00:21 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-20 17:00:21 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-22 13:27:16 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-22 13:27:16 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-22 13:27:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-22 13:27:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-22 13:30:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-22 13:30:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-22 13:30:16 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-22 13:30:16 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-22 13:35:12 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-22 13:35:12 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-22 13:35:12 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-22 13:35:12 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 17:36:42 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 17:36:42 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 17:36:42 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 17:36:42 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 18:36:50 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 18:36:50 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 18:36:50 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 18:36:50 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 18:41:01 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 18:41:01 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 18:41:01 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 18:41:01 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 18:44:44 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 18:44:44 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 18:44:44 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 18:44:44 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 18:49:11 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 18:49:11 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 18:49:11 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 18:49:11 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 18:53:29 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 18:53:29 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 18:53:29 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 18:53:29 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 19:00:12 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 19:00:12 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 19:00:12 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 19:00:12 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 19:06:37 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 19:06:37 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 19:06:37 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 19:06:37 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 19:08:41 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 19:08:41 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 19:08:41 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 19:08:41 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 19:15:25 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 19:15:25 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 19:15:25 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 19:15:25 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 19:21:20 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 19:21:20 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 19:21:20 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 19:21:20 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 19:26:02 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 19:26:02 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 19:26:02 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 19:26:02 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 19:35:48 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 19:35:48 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 19:35:48 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 19:35:48 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 19:36:42 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 19:36:42 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 19:36:42 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 19:36:42 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 19:46:59 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 19:46:59 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 19:46:59 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 19:46:59 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 19:52:01 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 19:52:01 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 19:52:01 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 19:52:01 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 19:54:06 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 19:54:06 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 19:54:06 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 19:54:06 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 20:17:23 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 20:17:23 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 20:17:24 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-27 20:17:24 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-28 11:58:52 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-28 11:58:52 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-28 11:58:52 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-28 11:58:52 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-28 12:04:59 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-28 12:04:59 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-28 12:04:59 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-28 12:04:59 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-28 12:22:54 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-28 12:22:54 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-28 12:22:54 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-28 12:22:54 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-28 12:24:44 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-28 12:24:44 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-28 12:24:44 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-28 12:24:44 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-28 12:27:40 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-28 12:27:40 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-28 12:27:40 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-28 12:27:40 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-28 12:30:45 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-28 12:30:45 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-28 12:30:45 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-28 12:30:45 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-28 12:37:19 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-28 12:37:19 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-28 12:37:19 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-28 12:37:19 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-28 12:40:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-28 12:40:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-28 12:40:16 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-28 12:40:16 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-28 12:49:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-28 12:49:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-28 12:49:24 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-28 12:49:24 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-28 13:34:12 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-28 13:34:12 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-28 13:34:12 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-28 13:34:12 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-29 11:18:31 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-29 11:18:31 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-29 11:19:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-29 11:19:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-29 11:19:00 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-29 11:19:00 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-29 12:28:35 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-29 12:28:35 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-29 12:28:35 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-29 12:28:35 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-29 12:28:35 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-29 12:28:35 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-29 12:28:40 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-29 12:28:40 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-29 12:28:40 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-29 12:28:40 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-29 18:03:52 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-29 18:03:52 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-29 18:05:54 database: close
org.h2.message.DbException: IO Exception: "java.io.EOFException"; "/home/acalabro/workspaceLearnPAd/learnpad/lp-simulation-environment/monitoring/data/glimpse.h2.db" [90031-176]
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.convertIOException(DbException.java:329)
	at org.h2.store.FileStore.readFully(FileStore.java:276)
	at org.h2.store.PageStore.readPage(PageStore.java:1326)
	at org.h2.store.PageStore.getPage(PageStore.java:750)
	at org.h2.store.PageStreamTrunk$Iterator.next(PageStreamTrunk.java:252)
	at org.h2.store.PageLog.free(PageLog.java:214)
	at org.h2.store.PageStore.compact(PageStore.java:499)
	at org.h2.engine.Database.closeOpenFilesAndUnlock(Database.java:1301)
	at org.h2.engine.Database.close(Database.java:1251)
	at org.h2.engine.Database.removeSession(Database.java:1122)
	at org.h2.engine.Session.close(Session.java:672)
	at org.h2.jdbc.JdbcConnection.close(JdbcConnection.java:383)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at com.onseven.dbvis.h.B.C.ā(Z:2117)
	at com.onseven.dbvis.h.B.F$A.call(Z:1369)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.h2.jdbc.JdbcSQLException: IO Exception: "java.io.EOFException"; "/home/acalabro/workspaceLearnPAd/learnpad/lp-simulation-environment/monitoring/data/glimpse.h2.db" [90031-176]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:344)
	... 23 more
Caused by: java.io.EOFException
	at org.h2.store.fs.FileUtils.readFully(FileUtils.java:347)
	at org.h2.store.FileStore.readFully(FileStore.java:274)
	... 20 more
06-29 18:05:54 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-29 18:05:54 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-29 18:06:25 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-29 18:06:25 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-29 18:08:36 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-29 18:08:36 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
2016-06-29 18:18:18 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "BPMN" not found; SQL statement:
SELECT absolute_bp_score FROM bpmn [42102-191]
06-29 18:24:44 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-29 18:24:44 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-29 18:24:44 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-29 18:24:44 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
2016-06-29 18:25:18 database: close
org.h2.message.DbException: IO Exception: "java.io.EOFException"; "/home/acalabro/workspaceLearnPAd/learnpad/lp-simulation-environment/monitoring/data/glimpse.h2.db" [90031-191]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convertIOException(DbException.java:330)
	at org.h2.store.FileStore.readFully(FileStore.java:275)
	at org.h2.store.PageStore.readPage(PageStore.java:1325)
	at org.h2.store.PageStore.getPage(PageStore.java:749)
	at org.h2.store.PageStreamTrunk$Iterator.next(PageStreamTrunk.java:251)
	at org.h2.store.PageLog.free(PageLog.java:213)
	at org.h2.store.PageStore.compact(PageStore.java:498)
	at org.h2.engine.Database.closeOpenFilesAndUnlock(Database.java:1356)
	at org.h2.engine.Database.close(Database.java:1286)
	at org.h2.engine.DatabaseCloser.run(DatabaseCloser.java:63)
Caused by: org.h2.jdbc.JdbcSQLException: IO Exception: "java.io.EOFException"; "/home/acalabro/workspaceLearnPAd/learnpad/lp-simulation-environment/monitoring/data/glimpse.h2.db" [90031-191]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 11 more
Caused by: java.io.EOFException
	at org.h2.store.fs.FileUtils.readFully(FileUtils.java:358)
	at org.h2.store.FileStore.readFully(FileStore.java:273)
	... 8 more
06-29 18:25:46 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-29 18:25:46 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-29 18:26:07 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-29 18:26:07 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-29 18:26:07 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-29 18:26:07 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-30 15:57:41 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-30 15:57:41 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-30 15:57:42 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-30 15:57:42 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-30 15:57:56 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-30 15:57:56 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-30 16:07:37 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-30 16:07:37 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-30 16:07:37 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-30 16:07:37 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-30 16:07:37 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
06-30 16:07:37 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-176]
